Sources/CartonCLI/Commands/TestRunners/BrowserTestRunner.swift

```diff
@@ -134,7 +134,7 @@ struct BrowserTestRunner: TestRunner {
   }
 
   func run() async throws {
-    defer { try httpClient.syncShutdown() }
+    defer { try? httpClient.syncShutdown() }
```

Comment:
Is there a specific reason for this change? If so, could you add a code comment explaining why `try?` had to be used?

Reply:
Without this change Xcode 13.4/CLI Swift 5.6.1 build fails with `Call can throw, but errors cannot be thrown out of a defer body`.
I do see that previous PR pass the build with same Xcode so can't really explain it. Don't mind reverting this and keeping this change local for now.